src/components/shared/eventcard/eventcard.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Day from 'components/shared/day';
+import Box from 'components/shared/box';
+import Text from 'components/shared/text';
+import Button from 'components/shared/button';
+
+import EventPlaceholderImage from 'images/event-placeholder.png';
+
+import ClockIcon from 'images/clock.svg';
+import MapMarkerIcon from 'images/map-marker.svg';
+
+import {
+  Article,
+  ImageWrapper,
+  Image,
+  RSVPBox,
+  IconList,
+  IconRow,
+  IconText,
+  IconLink,
+} from './eventcard.css';
+
+import { parse, format } from 'date-fns';
+
+const EventCard = ({ event }) => {
+  const { name, local_date, time, featured_photo, venue, short_link } = event;
+
+  const image = () => {
```

Comment:
Nit, this doesn't have to be a component, which gives pointless overhead. Can just assign the React node to the variable directly.

Reply:
Also nit, I think this would be cleaner just rendering the image inline below with src being the ternary featurrd_photo != null ? Your photo : placeholder.

Reply:
Done in e7b047d.